lib/clusters/iasZone.js

```diff
@@ -1,11 +1,61 @@
 'use strict';
 
 const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
 
 const ATTRIBUTES = {
+  zoneState: {
+    id: 0,
+    type: ZCLDataTypes.enum8({
+      notEnrolled: 0,
+      enrolled: 1,
+    }),
+  },
+  zoneType: {
+    id: 1,
+    type: ZCLDataTypes.enum8({
```

Comment:
According to the docs (and the long list of enum values) this should be enum16.

Reply:
fixed in next commti